antareslauncher/data_repo/data_repo_tinydb.py

```diff
@@ -90,6 +90,7 @@ def save_study(self, study: StudyDTO):
             self.db.update(new, tinydb.where(pk_name) == pk_value)
         else:
             logger.info(f"Inserting study '{pk_value}' in database: {new!r}")
+            new["antares_version"] = f"{new['antares_version']:2d}"
```

Comment:
it's not very safe to do this because we change the study object:
if it's used after this method call, antares_version is a string and not a StudyVersion anymore.
It would be more safe to copy the dict for example

Reply:
Ah okay I see, modifying `vars(study)` alters the `study` object. I didn't think about that. I'll do a deepcopy of the dict.